core/src/services/gcs/backend.rs

```diff
@@ -405,6 +407,50 @@ impl Accessor for GcsBackend {
         ))
     }
 
+    async fn copy(&self, from: &str, to: &str, _: OpCopy) -> Result<RpCopy> {
+        let source = percent_encode_path(&build_abs_path(&self.root, from.trim_end_matches("/")));
+        let dest = percent_encode_path(&build_abs_path(&self.root, to.trim_end_matches("/")));
+        let req_uri = format!(
+            "{}/storage/v1/b/{}/o/{}/rewriteTo/b/{}/o/{}",
```

Comment:
Is something not quite right here? I think it should be this:
https://cloud.google.com/storage/docs/json_api/v1/objects/copy